translate/cloud-client/translate_v3_batch_translate_text_with_glossary_test.py

```diff
@@ -12,18 +12,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import backoff
 import os
 import pytest
 import uuid
 import translate_v3_batch_translate_text_with_glossary
 import translate_v3_create_glossary
 import translate_v3_delete_glossary
+
+from google.api_core.exceptions import DeadlineExceeded, GoogleAPICallError, RetryError
+from google.cloud.exceptions import NotFound
 from google.cloud import storage
 
 PROJECT_ID = os.environ["GCLOUD_PROJECT"]
 GLOSSARY_INPUT_URI = "gs://cloud-samples-data/translation/glossary_ja.csv"
 
 
+@backoff.on_exception(
```

Comment:
Do you know how does it work when you wrap the fixture with `backoff`?

Can you just decorate an inner function as I did?

Reply:
Ahh, I see what I did wrong.